bucket/hwp-converter.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "2.0",
+    "description": "A tool which allows you to convert and open HWP files in Word as DOCX file",
+    "homepage": "https://www.microsoft.com/en-us/download/details.aspx?id=49152",
+    "license": "Proprietary",
+    "architecture": {
+        "32bit": {
+            "url": "https://download.microsoft.com/download/1/8/D/18D83826-2F32-4535-8C6A-686149B39DF8/hwpconverter_x86_en-us.exe#dl.7z",
+            "hash": "48d599393a0a15b9c81a453bfa2e8db5d546e4cb63045cc382629f4fa35db82e",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\hwpconverter.msi\" -DestinationPath \"$dir\\tmp\"",
+                    "Move-Item \"$dir\\tmp\\BATCHHWPCONV.EXE.x86\" \"$dir\\BatchHwpConv.exe\"",
+                    "Move-Item \"$dir\\tmp\\HWPCONV.EXE.x86\" \"$dir\\HwpConv.exe\"",
+                    "Remove-Item \"$dir\\files.cat\", \"$dir\\tmp\" -Recurse"
+                ]
+            }
+        },
+        "64bit": {
+            "url": "https://download.microsoft.com/download/1/8/D/18D83826-2F32-4535-8C6A-686149B39DF8/hwpconverter_x64_en-us.exe#dl.7z",
+            "hash": "1a6a69c00a60f0ba44527c345017e4aba309b3e0003ee463721913ab9c6d9302",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\hwpconverter.msi\" -DestinationPath \"$dir\\tmp\"",
+                    "Move-Item \"$dir\\tmp\\BATCHHWPCONV.EXE.x64\" \"$dir\\BatchHwpConv.exe\"",
+                    "Move-Item \"$dir\\tmp\\HWPCONV.EXE.x64\" \"$dir\\HwpConv.exe\"",
+                    "Remove-Item \"$dir\\files.cat\", \"$dir\\tmp\" -Recurse"
+                ]
+            }
+        }
+    },
+    "post_install": [
```

Comment:
What is this for?? Executing normal installer after own installation is not good idea.

Reply:
I've extracted a single binary from the MSI file and it works. But if you execute the normal installer, the installer will inject a hwp converter plugins to the installation directory of MS Word 2016 which is quite useful.

But injecting files in such manner is not portable at all and requires root permission, which is not quite scoop's way. So I just left a message for users who might want to run the installer manually.